doc/source/ray-air/doc_code/tf_starter.py

```diff
@@ -52,25 +52,11 @@ def train_func(config: dict):
 
     dataset = session.get_dataset_shard("train")
 
-    def to_tf_dataset(dataset, batch_size):
-        def to_tensor_iterator():
-            for batch in dataset.iter_tf_batches(
-                batch_size=batch_size, dtypes=tf.float32
-            ):
-                yield batch["x"], batch["y"]
```

Comment:
Our examples are over simplified with trivial column selection, but in reality user will need a lot of customization here, how will user be able to do batch level manipulation with the new `to_tf` change ?

Reply:
> how will user be able to do batch level manipulation with the new to_tf change ?

Wdym by batch-level manipulation?

Reply:
This should work for tabular and image workloads (so basically all of our examples). What're some other workloads we should consider?